org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/JavaSearchScope.java

```diff
@@ -52,9 +52,10 @@
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class JavaSearchScope extends AbstractJavaSearchScope {
 
-	private ArrayList elements;
+	private HashSet elements;
```

Comment:
```suggestion
	private Set<JavaElement> elements;
```

Reply:
I followed the existing style as closely as I could to create the minimal change.

Reply:
The line is already changed, so how does the change becomes more minimal with not using generics?